### PR TITLE
MYNEWT-681 rb-nano2 remove flash write protection in download scripts 

### PR DESCRIPTION
**Pull request by vrahane**
- removing write protection in download script by setting the write
  protection bits in the BPROT config registers. Also, setting the
  DISABLEINDEBUG register.

### CODE REVIEW

File: hw/bsp/rb-nano2/rb-nano2_download.sh [hunk 1]
```diff
@@ -37,6 +37,15 @@ if [ "$MFG_IMAGE" ]; then
     FLASH_OFFSET=0
 fi
 
+# We write the config registers for BPROT so that NVMC write
+# protection gets disabled for all blocks of the flash
+# We also write the DISABLEINDEBUG register so that, write
+# protection gets disabled in debug register by default
+EXTRA_GDB_CMDS="set {unsigned long}0x40000600=0;set {unsigned long}0x40000604=0;\
```
Comment (mkiiskila) [inline hw/bsp/rb-nano2/rb-nano2_download.sh]: Did you mean CFG_POST_INIT instead of EXTRA_GDB_CMDS?

Comment (vrahane) [inline hw/bsp/rb-nano2/rb-nano2_download.sh]: Sorry, my bad. I assumed there was a EXTRA_GDB_CMDS in the openocd script. CFG_POST_INIT goes to the OCD_CMD_FILE which is more of ocd commands than gdb commands. I think what is missing in the openocd script is EXTRA_GDB_CMDS which goes to the GDB_CMD_FILE.

**Comment by vrahane** [comment]
Changed it to use openocd commands instead of gdb commands. Also added the EXTRA_GDB_CMDS variable for future use.

File: hw/scripts/openocd.sh [hunk 2]
```diff
@@ -101,7 +101,9 @@ openocd_debug () {
         if [ ! -z "$RESET" ]; then
             echo "mon reset halt" >> $GDB_CMD_FILE
         fi
-	echo "FILENAME" $FILE_NAME >>out
+  echo "$EXTRA_GDB_CMDS" >> $GDB_CMD_FILE
```
Comment (mkiiskila) [inline hw/scripts/openocd.sh]: Indentation is wrong on this line. Are you using tab instead of space?

File: hw/scripts/openocd.sh [hunk 3]
```diff
@@ -101,7 +101,9 @@ openocd_debug () {
...
+  echo "FILENAME" $FILE_NAME >>out
```
Comment (mkiiskila) [inline hw/scripts/openocd.sh]: I don't see a reason for this line. Why is it here? Could it be removed as part of this commit?

**Pull request merged**